dropwizard-testing/src/main/java/io/dropwizard/testing/DropwizardTestSupport.java

```diff
@@ -61,6 +72,31 @@ public DropwizardTestSupport(Class<? extends Application<C>> applicationClass, S
         this.configPath = configPath;
         this.configOverrides = ImmutableSet.copyOf(firstNonNull(configOverrides, new ConfigOverride[0]));
         this.customPropertyPrefix = customPropertyPrefix;
+        explicitConfig = false;
+    }
+
+    /**
+     * Alternative constructor that may be used to directly provide Configuration
+     * to use, instead of specifying resource path for locating data to create
+     * Configuration.
+     *
+     * @since 0.9
+     *
+     * @param applicationClass Type of Application to create
+     * @param config Pre-constructed configuration object caller provides; will not
+     *   be manipulated in any way, no overriding
+     */
+    public DropwizardTestSupport(Class<? extends Application<C>> applicationClass,
+            C configuration) {
+        if (configuration == null) {
+            throw new IllegalArgumentException("Can not pass null configuration for explicitly configured instance");
+        }
+        this.applicationClass = applicationClass;
+        configPath = "";
+        configOverrides = Collections.emptySet();
```

Comment:
Maybe better to use `ImmutableSet.of()` in order to be consistent with other constructor?